boards/esp32-ethernet-kit-v1_0/Kconfig

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2020 HAW Hamburg
+# Copyright (c) 2020 Google LLC
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+config BOARD
+    default "esp32-ethernet-kit-v1_0" if BOARD_ESP32_ETHERNET_KIT_V1_0
+
+config BOARD_ESP32_ETHERNET_KIT_V1_0
+    bool
+    default y
+    select BOARD_COMMON_ESP32
+    select CPU_MODEL_ESP32_WROVER_B
+    select HAS_ESP_SPI_RAM
+    select HAS_PERIPH_ADC
+    select HAS_PERIPH_ETH
+    select HAS_PERIPH_I2C
+    select HAS_PERIPH_PWM
+
```

Comment:
`select HAS_PERIPH_SPI` has to be defined if module `esp_jtag` isn't used.

@leandrolanzieri It is possible to select a feature `XXX` in `Kconfig` if a module `YYY` is used.
```
select HAS_XXX if MODULE_YYY
```
Is it also possible to select a feature `XXX` if module `YYY` is not used?

Reply:
> It is possible to select a feature XXX in Kconfig if a module YYY is used.

Yes, exactly as you wrote. 

>Is it also possible to select a feature XXX if module YYY is not used?

Yes. To select it when the module `YYY` is not there:
```
select HAS_XXX if !MODULE_YYY
```